src/lib/editor.js

```diff
@@ -35,7 +35,7 @@ Editor.prototype = {
 
     this.editorCameraEl = document.createElement('a-entity');
     this.editorCameraEl.isEditor = true;
-    this.editorCameraEl.addEventListener('loaded', function(entity) {
+    this.editorCameraEl.addEventListener('loaded', function (entity) {
       this.DEFAULT_CAMERA = this.editorCameraEl.getObject3D('camera');
       this.initUI();
     }.bind(this));
```

Comment:
Yep, the editor.js is a legacy code in es5 used with three.js editor. I'll clean and convert to es6 in another PR